modules/contact-form/grunion-contact-form.php

```diff
@@ -1995,14 +1995,16 @@ function process_submission() {
 		 * @param string|array $to Array of valid email addresses, or single email address.
 		 */
 		$to = (array) apply_filters( 'contact_form_to', $to );
+		$reply_to_addr = $to[0]; // get just the address part before the name part is added
+
 		foreach ( $to as $to_key => $to_value ) {
 			$to[ $to_key ] = Grunion_Contact_Form_Plugin::strip_tags( $to_value );
+			$to[ $to_key ] = self::add_name_to_address( $to_value );
```

Comment:
Er, nevermind, now I remember why I didn't do that. We don't actually have a good source for the `$to` name, and it seems unlikely that users would take advantage of the ways it could be made available. It comes from these sources:

1. The `[contact-form to=""` parameter. The form builder will remove anything wrapped in `<` or `>` fields when inserting the shortcode, so it would break an address with a name part. I guess somebody could manually add it a name to the parameter, but that seems like a pretty small edge case, and it'd break if they ever used the form builder to modify the shortcode. `add_name_to_address()` does make sure that it only adds a name if it's passed a raw address, so it should work if anyone tries it, as long as they don't use the form builder.
1. `get_option( 'admin_email' )`. We could try to fetch any `WP_User` accounts with a matching email address, then pull that user's name, but it won't always work since `admin_email` is independent of user accounts. It wouldn't hurt anything, it could just fall back to using the address as the name if it didn't, but I'm not sure it's worth it. I'm happy to add it if you feel like it is, though.
1. The `contact_form_to` filter, which also seems like a small edge case. I don't think many people will interpret the filter docs as saying that a name part would be allowed, but it's possible. `add_name_to_address()` wouldn't modify the address if it were included, so this should already work.

Right now the `To` message header is the only place that's using `add_name_to_address()`, since the `From` and `Reply-To` headers are already built with names.